image/preseed/preseed_linux.go

```diff
@@ -536,15 +536,19 @@ func runUC20PreseedMode(opts *preseedOpts) error {
 	return nil
 }
 
-func Core20(chrootDir string) error {
-	popts, cleanup, err := prepareCore20Chroot(chrootDir)
+// Core20 runs preseeding of UC20 system prepared by prepare-image in prepareImageDir
+// and stores the resulting preseed artifact.tgz file in system-seed/systems/<systemlabel>/artifact.tgz.
```

Comment:
is that the name? or is preseed-artifact.tgz?

Reply:
It's actually preseed.tgz, thanks for carching.